src/transxchange/naptan.rs

```diff
@@ -266,10 +282,11 @@ mod tests {
 
         #[test]
         #[should_panic]
-        fn missing_coords() {
-            let csv_content = r#""StopAreaCode","Name"
-"010G0001","Bristol Bus Station"
-"010G0002","Temple Meads""#;
+        fn duplicate_id() {
```

Comment:
I wonder if this kind of test is useless. It is already tested in `collectionWithId::push` or `collectionWithId::new`

Reply:
So we discussed about that test with @datanel and decided to keep it. The concept of testing for duplicate in the collection and failing when there is a duplicate is a valid use case to test (but edge case, I agree).